jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-5.0.x/Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+node {
+    checkout scm
+    pipeline = load 'jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy'
+}
+
+pipeline.single_pipeline([
+        job_name: 'oss-5.0.x',
```

Comment:
Will this move to 5.1.x pretty quickly (perhaps in a month)? Would we want to make this a 5.x and change the regex to be `/5\.\d\.\d/`?